web/docs/Frontend Starters/Next.js/Next.js + Drupal/surrogate-key-based-caching.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Surrogate Key Based Cache Purging
+slug: next-drupal-surrogate-key-caching
+id: next-drupal-surrogate-key-caching
+sidebar_position: 7
+---
+
+## Before You Begin
+
+You should be familiar with the concept of surrogate key based caching and
+purging.
+
+See https://docs.fastly.com/en/guides/working-with-surrogate-keys for more
+information on working with surrogate keys.
+
+This guide uses Drupal with the
+[Pantheon Advanced Page Cache module](https://www.drupal.org/project/pantheon_advanced_page_cache)
+installed.
+
+## How It Works
+
+```mermaid
+sequenceDiagram
+	participant A as Client
+    participant B as Next.js + drupal-kit
+    participant C as Drupal
+    A->>B: Request a page that fetches from Drupal
+    B->>C: Add Pantheon-Debug header to request
+    C->>B: Surrogate-Key-Raw header included on response
```

Comment:
I think we'll also be changing this to just `Surrogate-Key` in the future. We'll need to update the docs then.